docs/notebooks/ensembles.ipynb

```diff
@@ -275,6 +275,188 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "chg_frac, pos_frac = ensembles.change_significance(fut, ref, test=None)\n",
+    "agreement = (pos_frac > 0.8) | (pos_frac < 0.2)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Here, `chg_frac` is the fraction of models showing significant change. Since `test = None`, this array is all ones, significance was not tested. `pos_frac` is the fraction of members showing positive change. Usually, when less than 80% of the members agree on the sign of the change, the region is hatched."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "fig, ax = plt.subplots(figsize=(6, 4))\n",
+    "mean_delta.plot.contourf(ax=ax)\n",
+    "agreement.plot.contourf(\n",
+    "    ax=ax,\n",
+    "    levels=[-0.5, 0.5, 1.5],\n",
+    "    cmap=\"none\",\n",
+    "    hatches=[\"///\", None],\n",
+    "    add_colorbar=False,\n",
+    ")\n",
+    "ax.legend(\n",
+    "    handles=[\n",
+    "        Rectangle((0, 0), 2, 2, fill=False, hatch=\"///\", label=\"Low model agreement\")\n",
+    "    ],\n",
+    "    bbox_to_anchor=(0.0, 1.1),\n",
+    "    loc=\"upper left\",\n",
+    ");"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Approach C: The advanced alternative.\n",
+    "\n",
+    "Here, we define three categories : robust change, conflicting change and no significant change or robust change. Change is said to be significant if the delta exceed an internal variability. The IPCC document suggests two different ways for computing this variability threshold:\n",
+    "\n",
+    "- $\\gamma = \\sqrt{2}\\cdot1.645\\cdot\\sigma_{20yr}$ where $\\sigma_{20yr}$ is the standard deviation of 20-year means, computed from non-overlapping periods in the pre-industrical control runs, after detrending with a quadratic fit.\n",
+    "- $\\gamma = \\sqrt{\\frac{2}{20}}\\cdot1.645\\cdot\\sigma_{1yr}$ where $\\sigma_{1yr}$ is the interanual standard deviation computed over the linearly detrended reference period.\n",
+    "\n",
+    "The first method involves data that is not always easily available. When such data is at hand, the xclim function could be used with a custom computation of the $\\gamma$ (`gamma`) threshold:\n",
+    "\n",
+    "```python\n",
+    "chg_frac, pos_frac = ensembles.change_significance(fut, ref, test='threshold', abs_thresh=gamma)\n",
+    "```\n",
+    "\n",
+    "The second method is more accessible and xclim implements it directly. However, `pos_frac` output of xclim is the fraction of the members showing _significant_ positive change, while the IPCC defines the agreement separately from the change significance. So we run the function twice."
```

Comment:
That's not super clear.

Reply:
What do you think of the apparent complexity of this ?  See main comment, but the double call comes from the our definition of `pos_frac` that is different from the IPCC's. This could be modified, but would be breaking.

Reply:
Here I'm not suggestion to change the code itself, just the explanations. In fact, You may not have to give so much information in the notebook. I suggest to focus on the method that's actually implemented (don't mention the first method), and explain how it differs from the IPCC's in a bit more sentences. 

The "So we run the function twice." is a bit cryptic.